.github/workflows/test-modified.yml

```diff
@@ -16,7 +16,7 @@ jobs:
   build:
     if: github.repository == 'Project-MONAI/tutorials'
     container:
-      image: nvcr.io/nvidia/pytorch:24.03-py3
+      image: nvcr.io/nvidia/pytorch:24.02-py3
```

Comment:
Hi @KumoLiu , @mingxin-zheng ,

Please note 1 potential issue that: 24.03 is our target version to support in MONAI Toolkit, if we stick to xformer's requirement, we will not be easy to upgrade PyTorch version for the future MONAI Toolkit.

Thanks.

Reply:
Hi @Nic-Ma , this change is to revert previous changes that attempt to update the container image in tutorial CI pipeline. The `xformers` requires a delicate env setup on cuda and torch version. As I noticed the tutorial CI uses OSS pytorch, I used the OSS xformer to test this notebook.